lib/index.js

```diff
@@ -93,14 +91,20 @@ function increaseDifficulty(overlord, world){
 }
 
 function checkLevel(){
-  var level = document.getElementById('level'); 
+  var level = document.getElementById('level');
   if(trayCollision.total >= 200 && trayCollision.total < 400 ){
     level.innerHTML = "<h3>Level 2</h3>";
   } else if (trayCollision.total >= 400 && trayCollision.total < 600 ){
     level.innerHTML = "<h3>Level 3</h3>";
   }
 }
 
+function checkGameEnders(){
+  if(trayCollision.total < 0 && gameRunning || trayCollision.total >= 600 && gameRunning || !gameRunning){
```

Comment:
@deborahleehamel 
I like the fact that you pulled this out into a separate method. Is there a reason that you wanted to put all of that inside the first if case versus multiple if/else if or even just separate if statements?